src/index.html

```diff
@@ -1,22 +1,450 @@
 <!doctype html>
-<html lang="en">
+<html
+  lang="en"
+  class="page"
+>
   <head>
     <meta charset="UTF-8" />
     <meta
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
-    <title>Landing Page</title>
+
+    <title>MYBIKE</title>
+
+    <link
+      rel="icon"
+      type="image/x-icon"
+      href="images/favicon.png"
+    />
+
     <link
       rel="stylesheet"
-      href="./styles/main.scss"
+      href="./styles/style.scss"
     />
+
     <script
       src="scripts/main.js"
       defer
     ></script>
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,400;0,500;0,700&display=swap"
+      rel="stylesheet"
+    />
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <body class="page__body">
```

Comment:
u duplicate the paddings u need, so instead of `120px`, u get `240px`. It completely breaks ur layout
Check [Loom video](https://www.loom.com/share/2fbea72aa2de40f193f8d1025f66edb8) and   [Figma](https://www.figma.com/design/NZQAIydtHo5QkINyGLHNcq/BIKE-New-Version?node-id=41317-96&t=IuAmCW51Ia7upl7D-0)